zeek/plugin/include/runtime-support.h

```diff
@@ -602,7 +603,7 @@ inline ::zeek::ValPtr to_val(const T& t, ::zeek::TypePtr target, std::string_vie
         else {
             // Field must be &optional or &default.
             auto attrs = rtype->FieldDecl(idx)->attrs;
-            if ( ! (zeek::compat::Attribute_Find(attrs, ::zeek::detail::ATTR_DEFAULT) ||
+            if ( ! (compat::Attribute_Find(attrs, ::zeek::detail::ATTR_DEFAULT) ||
```

Comment:
This change seems unnecessary and makes this use of a name from `zeek::compat` inconsistent with others in this file. Let's keep the previous, redundant, but clearer name.
```suggestion
            if ( ! (zeek::compat::Attribute_Find(attrs, ::zeek::detail::ATTR_DEFAULT) ||
```